src/WormholeJoin.sol

```diff
@@ -191,6 +192,8 @@ contract WormholeJoin {
             // amtToGenerate doesn't need overflow check as it is bounded by amtToTake
             vat.slip(ilk, address(this), int256(amtToGenerate));
             vat.frob(ilk, address(this), address(this), address(this), int256(amtToGenerate), int256(amtToGenerate));
+            (, uint256 art_) = vat.urns(ilk, address(this)); // Query the actual value as someone might have repaid debt without going through settle
```

Comment:
Why is this not just stored directly to `art` storage variable instead of using the `art_` stack variable?

Reply:
Yes, to be honest I don't have a good reason to not have done it directly this way before.
Changed!